yarn.lock

```diff
@@ -1682,24 +1687,21 @@ angular-bootstrap-temporary@0.11.0:
   resolved "https://registry.npmjs.org/angular-bootstrap-temporary/-/angular-bootstrap-temporary-0.11.0.tgz"
   integrity sha512-FSbxag9Jv8rlRk46+4l0x2aJ/3HBOaiBvVS+Rhiu1nEh2CIXz5EDq7c6OHnPrIdXS5KjIr64Zg3EgXP9bBACNQ==
 
-angular-mocks@1.5.11:
-  version "1.5.11"
-  resolved "https://registry.npmjs.org/angular-mocks/-/angular-mocks-1.5.11.tgz"
-  integrity sha512-eDVhiwiHlMEK6W47sIkiZh4We35Lqskg+G5rSTpUBPndwd2XM/uwuW+eklzOZhx+ODaFLuMsxeLY1h9rA3utIA==
-
-angular-ui-router@0.2.11:
-  version "0.2.11"
-  resolved "https://registry.npmjs.org/angular-ui-router/-/angular-ui-router-0.2.11.tgz"
-  integrity sha512-jmsIsFl03/pLqoWQp90qclreAYd27SXEEyGn69EZ4FZvFG+ZjAVI39QXe2fX5OYOLjQasgDMxI//emK0YaeE0w==
+angular-mocks@1.8.3:
+  version "1.8.3"
+  resolved "https://registry.yarnpkg.com/angular-mocks/-/angular-mocks-1.8.3.tgz#c0dd05e5c3fc014e07af6289b23f0e817d7a4724"
+  integrity sha512-vqsT6zwu80cZ8RY7qRQBZuy6Fq5X7/N5hkV9LzNT0c8b546rw4ErGK6muW1u2JnDKYa7+jJuaGM702bWir4HGw==
 
-angular@1.5.11:
-  version "1.5.11"
-  resolved "https://registry.npmjs.org/angular/-/angular-1.5.11.tgz"
-  integrity sha512-09DBOVVWo6rOQfdCBKGfEL0ZZIhf6P3fbeP3BU+ty5FU50DPiavVeDn8hQ4wXE8o4vKEEpzY1aRcRHJMCixWYA==
+angular-ui-router@1.0.30:
+  version "1.0.30"
+  resolved "https://registry.yarnpkg.com/angular-ui-router/-/angular-ui-router-1.0.30.tgz#de8a07fa1620fc5a979ee8448d0ded159792436f"
+  integrity sha512-8xMpxbOtCJbRnGR1fhbyZ5BhFXr3zs1L2ytXQiBACTJjot309QNFFCaWx0lDM2eVXp8qte4idigU9gwWdspmtQ==
+  dependencies:
+    "@uirouter/core" "6.0.8"
 
-angular@>=1.5.0, angular@^1.6.3:
+angular@1.8.3, angular@>=1.5.0, angular@^1.6.3:
   version "1.8.3"
-  resolved "https://registry.npmjs.org/angular/-/angular-1.8.3.tgz"
+  resolved "https://registry.yarnpkg.com/angular/-/angular-1.8.3.tgz#851ad75d5163c105a7e329555ef70c90aa706894"
   integrity sha512-5qjkWIQQVsHj4Sb5TcEs4WZWpFeVFHXwxEBHUhrny41D8UrBAd6T/6nPPAsLngJCReIOqi95W3mxdveveutpZw==
```

Comment:
Interesting to see two versions of angular in the lock file, one of which was already at 1.8.3. Is the app already using version 1.8.3 somehow?

Reply:
Checking `angular.version` in the console on workflow suggests not, as it gives:

```js
{ full: "1.5.11", major: 1, minor: 5, dot: 11, codeName: "princely-quest" }
```